src/Service/Health/HealthIndicatorStack.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Akondas\ActuatorBundle\Service\Health;
+
+class HealthIndicatorStack implements \JsonSerializable
```

Comment:
you don't want service that can be serialized, I was thinking about separate value object that is produced by this service, for example HealthStatus
```php
HealthIndicatorStack::health(): HealthStatus
```

and then in the controller:
```php
        $health = $this->healthIndicatorStack->health();

        return new JsonResponse($health, $health->isUp() ? 200 : 503);
```

the names are WIP, maybe you'll find better

Reply:
Thanks for the review
I've named it `HealthStack` but i'm open for suggestions